examples/macroquad-example/src/main.rs

```diff
@@ -0,0 +1,54 @@
+use std::f32::consts::PI;
+
+use macroquad::prelude::*;
+use macroquad::math;
+use lilt::*;
+
+#[macroquad::main("BasicShapes")]
+async fn main() {
+    let time = get_time() as f32 * 1000.;
+    let animation1: Animated<bool, f32> = Animated::new(false)
+                        .duration(2000.)
+                        .easing(Easing::EaseInOutQuint)
+                        .repeat_forever()
+                        .auto_reverse()
+                        .auto_start(true, time);
+
+    let animation2: Animated<bool, f32> = Animated::new(false)
+                        .duration(2000.)
+                        .easing(Easing::Linear)
+                        .repeat_forever()
+                        .auto_start(true, time);
+
+    loop {
+        let time = get_time() as f32 * 1000.;
+        clear_background(LIGHTGRAY);
+
+        let rect_left = animation1.animate_bool(100., screen_width() - 200.0, time);
+        let rotation = animation2.animate_bool(0., 2., time);
+
+        let ox = 300.;
+        let oy = 300.;
```

Comment:
What's the o prefix here?

Reply:
it is the rotation center, I'll find a more explicit way.